src/Apps/Artist/Routes/AuctionResults/Components/AuctionFilters/YearCreated.tsx

```diff
@@ -74,6 +75,14 @@ export const YearCreated: React.FC = () => {
           selected={`${createdBeforeYear}`}
         />
       </Flex>
+      <Checkbox
+        selected={allowEmptyCreatedDates}
+        onSelect={(allowEmpty: boolean) => {
+          filterContext.setFilter("allowEmptyCreatedDates", allowEmpty)
+        }}
+      >
+        {"Include unspecified dates"}
```

Comment:
```suggestion
        "Include unspecified dates"
```
You don't necessarily need the braces if it's just a static string